runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -250,4 +250,33 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 	}
+
+	private LootTrackerItemEntry[] buildEntries(final Collection<ItemStack> itemStacks)
+	{
+		return itemStacks.stream().map(itemStack ->
+		{
+			final ItemComposition itemComposition = itemManager.getItemComposition(itemStack.getId());
+			final int realItemId = itemComposition.getNote() != -1 ? itemComposition.getLinkedNoteId() : itemStack.getId();
+			final ItemPrice itemPrice = itemManager.getItemPrice(realItemId);
```

Comment:
This call is also safe on any thread, so probably I would move that back to the UI aswell.

Reply:
Yeah, but then I would need to set the "realItemId" instead to the entry and then send itemManager back there. I dont see any benefit in doing either, so why do you prefer that?